import-service/src/services/s3Service.js

```diff
@@ -0,0 +1,64 @@
+import csvParser from 'csv-parser';
+
+export const getSignedUrl = async (s3, params) => {
+  try {
+    const signedUrl = await s3.getSignedUrl('putObject', params);
+
+    return signedUrl;
+  } catch (error) {
+    throw new Error(`${error}: Unable to upload file`);
+  }
+};
+
+export const moveParsedFile = async (s3, params, record) => {
+  const uploaded = 'uploaded';
+  const parsed = 'parsed';
+  const copyParams = {
+    Bucket: params.Bucket,
+    CopySource: `${params.Bucket}/${params.Key}`,
+    Key: params.Key.replace(uploaded, parsed),
```

Comment:
What if a file has "uploaded" in its name?

Reply:
Sorry, could you be more specific about it?

Reply:
If you'll try to import a file with the name "uploaded.csv", will it work correctly?

Reply:
Yes it will work correctly. Method replace will change only the first match if we use it without key g(global).